Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -507,14 +509,14 @@ def __validate_marketplace_compatibility(self,
 
         match self.marketplace:
             case MarketplaceVersions.MarketplaceV2:
-                if content_item_marketplaces != (self.marketplace,):
-                    # marketplacev2 must be the only value in order to be collected
+                # For XSIAM machines we collect tests that have not xsoar marketplace.
+                # Tests for the packs that has only mpv2, or mpv2 and xpanse marketplaces,
+                # will run on xsiam machines only.
+                if MarketplaceVersions.XSOAR in content_item_marketplaces:
```

Comment:
If content has `xpanse` only, the existing code would consider it as XSIAM compatible. Is it so?
If not, please accept the following suggestion 

```suggestion
                if (MarketplaceVersions.MarketplaceV2 not in content_item_marketplaces) or (MarketplaceVersions.XSOAR in content_item_marketplaces):
```